makefile

```diff
@@ -0,0 +1,24 @@
+# By default, this makefile will run everything inside a Docker container.
+# If you're a developer and want to run everything with your local `cargo`:
+# - Creates a `.env` file in the same folder as this `makefile`
+# - Add a line in `.env` with `LOCAL_DEV=true`
+-include .env
+
+ifeq ($(LOCAL_DEV),true)
+	SH=sh -c
+else
+	SH=docker run --rm --user "$(id -u)":"$(id -g)" --volume "${PWD}":/tmp/skip_error --workdir /tmp/skip_error rust:latest sh -c
+endif
+
+fmt: format
+
+format:
+	${SH} "rustup component add rustfmt && cargo fmt --all -- --check"
+
+clippy: lint
+
+lint:
+	${SH} "rustup component add clippy && cargo clippy --workspace --all-features -- --warn clippy::cargo --allow clippy::multiple_crate_versions --deny warnings"
+
+test:
+	${SH} "cargo test"
```

Comment:
harmonize with clippy (features, workspaces, ...)

Reply:
Yep, I did that, just forgot to push. Should be ok now.